.eslintrc.json

```diff
@@ -36,24 +44,64 @@
     }
   ],
   "rules": {
-    "id-length": "off",
-    "no-magic-numbers": "off",
+    "id-length": "warn",
+    "no-magic-numbers": "warn",
     "jest/no-jasmine-globals": "off",
     "jest/no-identical-title": "warn",
     "jest/no-done-callback": "warn",
     "jest/no-disabled-tests": "warn",
     "jest/no-conditional-expect": "warn",
     "sort-keys-fix/sort-keys-fix": "warn",
     "jsx-a11y/media-has-caption": "warn",
-    "jsx-a11y/no-noninteractive-tabindex": "off",
-    "react/react-in-jsx-scope": "off",
-    "react/no-unknown-property": ["error", {"ignore": ["css", "transform-origin"]}],
+    "jsx-a11y/no-noninteractive-tabindex": "warn",
+    "react/no-unknown-property": ["error", {"ignore": ["css"]}],
+    "react/react-in-jsx-scope": "warn",
```

Comment:
Humm I think this one can stay to `off`, isn't it? I think we have valid scenario for disabling this for accessibility

Reply:
you are right